cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -499,12 +499,13 @@ func (g *genConversion) preexists(inType, outType *types.Type) (*types.Type, boo
 
 func (g *genConversion) Init(c *generator.Context, w io.Writer) error {
 	sw := generator.NewSnippetWriter(w, c, "$", "$")
-	sw.Do("func init() {\n", nil)
-	if g.targetPackage == apiPackagePath {
-		sw.Do("if err := Scheme.AddGeneratedConversionFuncs(\n", nil)
-	} else {
-		sw.Do("if err := api.Scheme.AddGeneratedConversionFuncs(\n", nil)
-	}
+	sw.Do("func AddGeneratedConversionFuncs(scheme *$.|public$) {\n",
```

Comment:
I'm pretty sure Clayton was doing similar changes - maybe this PR is not yet merged.